app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+    def index
+        @chef = Chef.find(params[:chef_id])
+        @ingredients = Chef.unique_ingredients(@chef.id)
+    end
+end
```

Comment:
For any nested resource routes you are going to want to send it to a joins controller. So instead of sending the request to the `IngredientsController`, because we are focused on ingredients for a particular chef, we want to send this to a controller that supports that. So we want to send requests to the `ChefIngredientsController`

---

For the `unique_ingredients` method, this could have been an instance method, and therefore you wouldn't need to pass in a chef.